src/Utils/OrientationUtils.js

```diff
@@ -173,6 +202,26 @@ export default {
         };
     },
 
+    /**
+     * Set the quaternion to correct for the
+     * <a href="https://geodesie.ign.fr/contenu/fichiers/documentation/algorithmes/alg0060.pdf">meridian convergence</a>
+     * of the East North Up (ENU) frame to the Lambert Conformal Conic (LCC) frame.
+     * This is a generally small rotation around Z.
+     *
+     * @param {Projection} proj the lcc projection (may be parsed using proj4)
+     * @param {number} proj.lat0 - the latitude of origin
+     * @param {number} proj.long0 - the longitude of the central meridian
+     * @param {Coordinates} [coordinates]  coordinates the origin of the local East North Up
+     * (ENU) frame
+     * @param {THREE.Quaternion} [target=new THREE.Quaternion()] output Quaternion
+     * @return {FunctionOrQuaternion} The target quaternion if coordinates is defined, otherwise, a function to compute it from coordinates.
+    */
+    quaternionToLCC(proj, coordinates, target = new THREE.Quaternion()) {
+        if (coordinates) { return this.quaternionToLCC(proj)(coordinates, target); }
+        const fromLCC = this.quaternionFromLCC(proj);
+        return (coordinates, target = new THREE.Quaternion()) => fromLCC(coordinates, target).conjugate();
```

Comment:
I prefer my way, to make the code self-documenting (otherwise you have to lookat fromLCC defaults to know what's the defaults of the returned function

Reply:
Ok, we could see that in a future re-factorization.